Makefile

```diff
@@ -4,11 +4,17 @@ PYTHON = $(shell which python3 || which python)
 export LANG
 
 
-.PHONY:init
+.PHONY:
```

Comment:
The original syntax here was correct: https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html
That is, we for all of the phony targets, we should include the target as a "dependency" of .PHONY.